src/app/shared/components/archive-small/archive-small.component.ts

```diff
@@ -29,10 +29,13 @@ export class ArchiveSmallComponent implements OnInit, OnChanges {
   @Output() editClick = new EventEmitter<any>();
 
   @Input() showAccept = false;
+  @Input() showAcceptMobile = false;
   @Input() acceptText = 'Accept';
   @Input() acceptIcon = 'check';
   @Output() acceptClick = new EventEmitter<any>();
 
+  @Input() showDefault = false;
```

Comment:
This is a change in behavior, right? If I understand correctly, the default should either be `true`, or we should update the non-onboarding call sites of this component to pass `showDefault = true`.